app/models/dream.rb

```diff
@@ -0,0 +1,3 @@
+class Dream < ApplicationRecord
+  belongs_to :user
```

Comment:
needs a has_many :bookings 
and has_many users through bookings
(Deji popped by just as we were looking at it 👍 )

Reply:
Added! Thanks